cornellGrading/cornellInterface.py

```
@@ -0,0 +1,104 @@
+import csv
+
+from sys import platform
+
+if platform == "linux" or platform == "linux2" or platform == "darwin":
+    from bullet import Bullet
+elif platform == "win32":
+    from consolemenu import *
+    from consolemenu.items import *
```

Should probably be a general `else` here in case of a different platform, which should raise an 'unsupported' error.